extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.css

```diff
@@ -0,0 +1,12 @@
+amp-story-audio-sticker {
+  z-index: 3 !important;
+  pointer-events: visible !important;
+
+  position: absolute;
+  max-width: 100%;
+  height: auto;
+}
+
+amp-story:not([muted]) amp-story-audio-sticker {
+  display: none  !important;
+}
```

Comment:
Was this not supposed to happen after 4 seconds? Also, nit: add newline at the end of the file